api/AudioParam.json

```diff
@@ -61,25 +61,25 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": false
             },
             "ie": {
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "55"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "48"
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Given the other mistake with Safari I checked this one, and `'automationRate' in AudioParam.prototype` is false in Safari 13.1.